src/hs111.jl

```diff
@@ -20,7 +20,7 @@ function hs111(args...)
 
   c = [-6.089, -17.164, -34.054, -5.914, -24.721, -14.986, -24.100, -10.708, -26.662, -22.179]
 
-  @variable(nlp, x[i=1:10], start=-2.3)
+  @variable(nlp, x[i=1:10], lowerbound = -100, start = -2.3, upperbound = 100)
```

Comment:
Sorry to be difficult, but for such simple bounds it would be more readable to write
```julia
@variable(nlp, -100 ≤ x[i=1:10] ≤ 100, start = -2.3)
```
See http://www.juliaopt.org/JuMP.jl/0.14/refvariable.html#variables. Does that work?